barretenberg/cpp/src/barretenberg/vm/generated/AvmMini_composer.test.cpp

```diff
@@ -35,29 +35,36 @@ TEST_F(AvmMiniTests, basic)
 
     trace_builder.callDataCopy(0, 3, 2, std::vector<FF>{ 45, 23, 12 });
 
-    trace_builder.add(2, 3, 4);
-    trace_builder.add(4, 5, 5);
-    trace_builder.add(5, 5, 5);
-    trace_builder.add(5, 6, 7);
+    //           Memory layout:    [0,0,45,23,12,0,0,0,....]
```

Comment:
Could you add comments that would explain what you are testing? Documenting intentions is helpful further down the line, when somebody will need to support this code